docs/appendix/apis/data-client.md

```diff
@@ -22,7 +22,7 @@ aliases:
   - /build/program/apis/data-client/
 ---
 
-The data client API allows you to upload and retrieve data to and from the [Viam app](https://app.viam.com).
+The data client API allows you to upload and retrieve [data](/services/data/) to and from the [Viam app](https://app.viam.com).
```

Comment:
technically untrue; the data client API allows you to upload and retrieve data to and from the Viam Cloud, which is accessible via the Viam App or any other client that interfaces w/ the Viam Cloud.

core distinction is that https://app.viam.com/ is just a wrapper over our backend in the same way that any other client would be. given that a key part of our product experience is offering SDKs that let users build their own interfaces, i think this is an important distinction to be clear on.

Reply:
this is the language we use in static/include/services/apis/generated/data_manager-table.md which is more appropriate:

"
| [`Sync`](/services/data/capture-sync/#sync) | **Sync data stored on the machine to the cloud.** | <p class="center-text"><i class="fas fa-times" title="no"></i></p> |"

Reply:
This API does a whole lot, not just sync https://docs.viam.com/appendix/apis/data-client/#api

Reply:
i wasn't suggesting that all the API does is sync, i was offering that language used as an example of how we can describe where data sync syncs the data

(in this instance we say "upload to and from Viam app", in the pasted reference we say "to the cloud") subtle diff, nbd either way.  

Also for what it's worth I spoke incorrectly; as the backend is a monorepo w/ the web client, technically we are uploading to "app" (referring to the github repo). IMO it's confusing to link to www.app.viam.com as it makes me think the web-client.  My 2c would be if we're going to say "upload to the Viam app" and link www.app.viam.com, we maybe specify cloud? ("upload to the Viam cloud").

Reply:
Oh I thought you were suggesting we say "sync" in place of "upload and retrieve." Well I did change it to cloud so that part is addressed!